### DISCUSSION TITLE
[Feature Request] Option to open menu with left click instead of hovering

### DISCUSSION THREAD

**Discussion by syntaxterror9** [zanllp/infinite-image-browsing#533] [Ideas]
Is it possible to add this option? This would mimic the familiar behavior we use in civitai.com to open the image menus/info windows.
thx!

**Comment by zanllp** [comment]
Which menu are you referring to? Could you please provide a screenshot for me to see?

**Comment by syntaxterror9** [comment]
you're right, I forgot to mention which menu :)

it's the three-dots pop up menu on the right-top corner of the preview images:

![iib-menu1](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/73241360/ebd68069-bdd8-4ca1-a4ba-0e1d3c54df9c)
![iib-menu2](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/73241360/db0b4ba5-58cf-41d4-ad21-d7e0a80a560d)

**Comment by zanllp** [comment]
You can right-click the mouse to perform this action.

https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/770036ec-26b1-4622-b0d9-c446bf3654ab

**Comment by zanllp** [comment]
However, the method you mentioned should also be able to achieve the same result. I will try to incorporate this feature later
